src/mediaembedediting.js

```diff
@@ -79,7 +79,9 @@ export default class MediaEmbedEditing extends Plugin {
 					name: 'youtube',
 					url: [
 						/^youtube\.com\/watch\?v=([\w-]+)/,
+						/^m\.youtube\.com\/watch\?v=([\w-]+)/,
```

Comment:
Can't we have `/^(m\.)?youtube\.com\/watch\?v=([\w-]+)/` instead?

Reply:
Only if we change `html()` callback: `const id = match[ 2 ] || match[ 1 ];`

Reply:
`/^(?:m\.)?youtube\.com\/watch\?v=([\w-]+)/`